hydro_deploy/hydro_cli/src/lib.rs

```diff
@@ -38,49 +28,6 @@ fn cleanup_runtime() {
     drop(TOKIO_RUNTIME.write().unwrap().take());
 }
 
-struct TokioRuntime {}
-
-impl pyo3_asyncio::generic::Runtime for TokioRuntime {
-    type JoinError = tokio::task::JoinError;
-    type JoinHandle = tokio::task::JoinHandle<()>;
-
-    fn spawn<F>(fut: F) -> Self::JoinHandle
-    where
-        F: Future<Output = ()> + Send + 'static,
-    {
-        TOKIO_RUNTIME
-            .read()
-            .unwrap()
-            .as_ref()
-            .unwrap()
-            .spawn(async move {
-                fut.await;
-            })
-    }
-}
-
-tokio::task_local! {
-    static TASK_LOCALS: OnceCell<TaskLocals>;
-}
-
-impl pyo3_asyncio::generic::ContextExt for TokioRuntime {
-    fn scope<F, R>(locals: TaskLocals, fut: F) -> Pin<Box<dyn Future<Output = R> + Send>>
-    where
-        F: Future<Output = R> + Send + 'static,
-    {
-        let cell = OnceCell::new();
-        cell.set(locals).unwrap();
-
-        Box::pin(TASK_LOCALS.scope(cell, fut))
-    }
-
-    fn get_task_locals() -> Option<TaskLocals> {
-        TASK_LOCALS
-            .try_with(|c| c.get().cloned())
-            .unwrap_or_default()
-    }
-}
```

Comment:
Only part I'm not sure about is removing this and using `pyo3_async_runtimes::tokio` instead of `pyo3_async_runtimes::generic`, not sure exactly what the difference is

Reply:
Ah so it's been a long time since I touched this code but basically there were many issues and panics happening when you Ctrl-C a deploy script because the Tokio runtime would be immediately dropped instead of staying alive long-enough for cleanup to happen.

I think this code has rotted quite too long and would be reasonably supportive of just removing it at this point... If we decide we want to bring it back later it won't be too difficult